journal/modernized/24-chapter-21.adoc

```diff
@@ -601,7 +601,7 @@ I opened; it was found to contain the address that follows,
 as literally translated by Samuel Wilson.
 
 [.signed-section-context-open]
-'`Tea-hu-poo, of June, 1835.
+Tea-hu-poo, of June, 1835.
```

Comment:
Removal of the apostrophe is good here, but I noticed the "of June" looked weird, and searched the original, and I think this should read:

`Tea-hu-poo, 26th of June, 1835.`